samples/extensions/sparse_image/README.md

```diff
@@ -0,0 +1,85 @@
+<!--
```

Comment:
The readme should use asciidoc (.adoc) instead of markdown. 

We have switched from markdown to asciidoc, so we can integrated the sample readmes into the vulkan documentation site. 

You can use online tools like https://pandoc.org/try/ to easily convert.